docs/docs/guides/response/django-pydantic.md

```diff
@@ -174,3 +174,36 @@ def modify_data(request, pk: int, payload: PatchDict[GroupSchema]):
 ```
 
 in this example the `payload` argument will be a type of `dict` only fields that were passed in request and validated using `GroupSchema`
+
+### Choices and Enums
+
+!!! info
+    Requires Django 5.0+ and Python 3.11+
+
+By default, ModelSchema, will ignore `choices` on a Model Field, and the schema will use
```

Comment:
Ignoring choices by default is a bit of an unexpected behavior. What is the reasoning for this?

Reply:
The default behavior of Shinobi/Ninja as it exists currently is to ignore `.choices`, rather, it has no support for it at all. This PR provides full support but requires a Mixin class be added (and Django 5+ and Python 3.11+). The other PR provides baseline/fallback support.